requirements.txt

```diff
@@ -3,6 +3,7 @@ h11
 
 # Optional
 httptools
+PyYAML>=5.1
```

Comment:
why not the latest ?

Reply:
My reasoning was that this was the least restrictive modern version of the library. I would expect that the latest version would be installed in the absence of other version specifications from other required libraries on the system.

Reply:
should i update this to the latest?

Reply:
lgtm